classes/02class/exercises/c02-network05/ramiray/README.md

```diff
@@ -0,0 +1,58 @@
+# c02-network05
+
+## Commands Execution Output
+
+- Commands for creating the internet gateway:
+```
+## Creating IGW:
+
+aws ec2 create-internet-gateway
+
+aws ec2 create-internet-gateway
+{
+    "InternetGateway": {
+        "Tags": [], 
+        "Attachments": [], 
+        "InternetGatewayId": "igw-0b50e51e2cf01b373"
+    }
+}
+
+## Naming the igw as "devopsacademy-igw":
+
+aws ec2 create-tags --resources igw-0b50e51e2cf01b373 --tags Key=Name,Value=devopsacademy-igw
+
+## Attaching the IGW to our VPC :
+
+aws ec2 attach-internet-gateway --internet-gateway-id igw-0b50e51e2cf01b373 --vpc-id vpc-0351acfbc7aed9c1f
+
+## Viewing the output :
+
+aws ec2 describe-internet-gateways --region ap-southeast-2 --filters Name=attachment.vpc-id,Values=vpc-0351acfbc7aed9c1f
+{
+    "InternetGateways": [
+        {
+            "Tags": [
+                {
+                    "Value": "devopsacademy-igw", 
+                    "Key": "Name"
+                }
+            ], 
+            "Attachments": [
+                {
+                    "State": "available", 
+                    "VpcId": "vpc-0351acfbc7aed9c1f"
+                }
+            ], 
+            "InternetGatewayId": "igw-0b50e51e2cf01b373"
+        }
+    ]
+}
+```
+
+- Any extra challenges faced?
+
+Its not a challenge but it would be better if we have a single command to do all at once. 
```

Comment:
No worries, someone told me that we are going to learn a nice tool to help us.
But now it is a good time to understand how the commands works and how they are linked with the AWS resources.